demo/radar.html

```diff
@@ -625,11 +644,68 @@
 					"truthLayer": lanesLayer
 				});
 			});
+
+			if (visualizationMode == "aptivLanes") {
+
+				const laneNum = [1,2,3];
+				const laneDirection = ["EB", "WB"];
+				const supplierNum = [1,2,3];
+				let filename, datasetName;
+
+				for (let s of supplierNum) {
+					for (let d of laneDirection) {
+						for (let n of laneNum) {
+							try {
+
+								let layerName = `Supplier${s}_${d}_Lane${n}`;
+								filename = `Supplier${s}_${d}_Lane${n}.fb`;
+
+								await loadLanes(s3, bucket, name, filename, s, (laneGeometries) => {
+									let lanesLayer = new THREE.Group();
+									lanesLayer.name = layerName;
+									for (let ii=0, len=laneGeometries.all.length; ii<len; ii++) {
+										lanesLayer.add(laneGeometries.all[ii]);
+									}
+									viewer.scene.scene.add(lanesLayer);
+									viewer.scene.dispatchEvent({
+										"type": "map_provider_layer_added",
+										"mapLayer": lanesLayer
+									});
+								});
+							} catch (e) {
+								console.log(`Couldn't load ${filename}: ${e}`);
+							}
+						}
+					}
+				}
+			}
+
+			// Load Comparison Dataset (hardcoded to one for now)
+			debugger;
```

Comment:
@vrajur-nd can you remove this when you get a chance?

Reply:
branch was already deleted so addressed in separate PR